src/libs/seedData.ts

```diff
@@ -6,14 +6,12 @@ export default () => {
     userRepository.count()
         .then(res => {
             // tslint:disable-next-line: no-shadowed-variable
-            if ( res => {
-                 console.log('Seeding Data in progress');
-            }) {
-                console.log('');
-            }
-        });
-    console.log('Seeding Data in progress!');
+            if ( res === 0) {
+                console.log('Seeding Data in progress');
+                userRepository.create({ name: 'Head Trainer', role: 'head-trainer', email: 'head.trainer@successive.tech', password: 'head@123'}, undefined);
```

Comment:
you can use below mention code :
`userRepository.create(
  {
    name: 'Head Trainer',
    role: 'head-trainer',
    email: 'head.trainer@successive.tech',
    password: 'head@123'
  },
  undefined)`

instead of 
`userRepository.create({ name: 'Head Trainer', role: 'head-trainer', email: 'head.trainer@successive.tech', password: 'head@123'}, undefined);`